compiler/rustc_mir_build/src/errors.rs

```diff
@@ -33,6 +33,11 @@ pub(crate) struct CallToDeprecatedSafeFnRequiresUnsafe {
 #[derive(Subdiagnostic)]
 #[multipart_suggestion(mir_build_suggestion, applicability = "machine-applicable")]
 pub(crate) struct CallToDeprecatedSafeFnRequiresUnsafeSub {
+    pub(crate) indent: String,
+    #[suggestion_part(
+        code = "{indent}// FIXME: Audit that the environment access only happens in single-threaded code.\n"
```

Comment:
The lint text is specific to particular functions, but the attribute is not. So this is in conflict with https://github.com/rust-lang/rust/pull/125970.

Reply:
Works for me. Maybe I can also do it this week-end.